### PR TITLE
Create physical_entity.md

### PR DESCRIPTION
**Pull request by cgendreau**

### CODE REVIEW

**Comment by cboelling** [comment]
So how is the process of reflecting e.g. today's discussion in this merge request - are additional changes made directly to physical_entity.md to accommodate them?

**Comment by cgendreau** [comment]
Yes, I just updated the document in this Pull Request

File: component_descriptions/physical_entity.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Loans and Transactions
+* Physical storage location(s)
+
+### Potential fields
+* Import Permit Tracking (CFIA for AAFC)
+* Field number or Other IDs of all sorts
+* Substrate type, air/water/soil
+* Potentially a History of State of the object - frozen/ dead/ in water/etc… (Link to preparation and protocol)
+* Amount? weight/ countable unit (3 parts of something) ( Talk to Falko for more)
+* Contaminated? Or another better term for this.
```
Comment (falkogloeckler) [inline component_descriptions/physical_entity.md]: As of the discussion on 16th Sept. this should be one (of potentially many) values of the field status, right?

File: component_descriptions/physical_entity.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* Date status changed
+  * State = destroyed
```
Comment (falkogloeckler) [inline component_descriptions/physical_entity.md]: This should not be binary
// todo: add more like "consumed", "lost", ...

**Review by falkogloeckler — APPROVED** [review]

File: component_descriptions/physical_entity.md (hunk 1 above)
Comment (cgendreau) [reply component_descriptions/physical_entity.md]: You mean Contaminated should be a value of another field representing the status like "Potentially a History of State of the object" ?

Comment (falkogloeckler) [reply component_descriptions/physical_entity.md]: Yes

Comment (cgendreau) [reply component_descriptions/physical_entity.md]: Made some adjustments

**Review by falkogloeckler — APPROVED** [review]
The values of states need some more discussion in order to align with preparation (for example: Does "frozen" mean the preparation type here or just the decontamination method?) 
But as the peparation component is still to be created, let's keep this jn mind for later.

**Comment by cgendreau** [comment]
Indeed, I will change it so it doesn't look like a final list

**Pull request merged**